### ISSUE TITLE
PSR-0 Deprecation Notice

### ISSUE THREAD

**Issue by darwin-focus** [rap2hpoutre/laravel-log-viewer#212]
Hello,

I am rather new to submitting these issues. If I have made an error or missed something I am sorry. Please let me know so I can fix it.

Thank you in advance.

### Issue Summary
I receive Deprecation Notice when pulling the package down with composer

```
Deprecation Notice: Class Rap2hpoutre\LaravelLogViewer\BaseController located in ./vendor/rap2hpoutre/laravel-log-viewer/src/controllers/LogViewerController.php does not comply with psr-0 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class Rap2hpoutre\LaravelLogViewer\BaseController located in ./vendor/rap2hpoutre/laravel-log-viewer/src/controllers/LogViewerController.php does not comply with psr-0 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class Rap2hpoutre\LaravelLogViewer\LogViewerController located in ./vendor/rap2hpoutre/laravel-log-viewer/src/controllers/LogViewerController.php does not comply with psr-0 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:185
```

### Steps to Reproduce
Install Composer version Composer version 1.10-dev
pull package `"rap2hpoutre/laravel-log-viewer": "^1.3"`
run composer install

Any other information you want to share that is relevant to the issue being reported. Especially, why do you consider this to be a bug? What do you expect to happen instead?

I expect no warnings or errors.

Technical details:
PHP Version: 7.3.3

**Comment by elminson** [comment]
I will take a look on this

**Comment by darwin-focus** [comment]
Any updates?

**Comment by aglipanci** [comment]
Any update here?

**Comment by rap2hpoutre** [comment]
Thank you! Fixed vthanks to @aglipanci, available in v1.5.0!
